engine/src/test/scala/org/scalarules/dsl/core/DatumTestsBerekening.scala

```diff
@@ -0,0 +1,23 @@
+package org.scalarules.dsl.core
+
+import org.scalarules.dsl.core.DatumTestGlossary._
+import org.scalarules.dsl.nl.grammar._
+
+class DatumTestsBerekening extends Berekening (
+
+  Gegeven (DatumA < "01-01-2015".datum)
+    Bereken KleinerDan is "success"
```

Comment:
Do you mean the `<` operator, or the name of the facts in the tests?

The operators are the ones already supported by the DSL and they will remain so. Although I do suggest we add specific date-related words to the DSL as 'aliases' for the mathematical operators.

The names of the facts, I'll concede and rename those to more appropriate names for date related handling. 👌